docs/content/intro/quickstart.md

```diff
@@ -0,0 +1,318 @@
+---
+title: A Brigade Quickstart
+description: A Brigade Quickstart.
+section: intro
+---
+
+## Install Brigade
+
+The easiest way to install Brigade into your Kubernetes cluster is to install it using [Helm](https://helm.sh/), the Kubernetes Package Manager.
+
+```bash
+# add Brigade chart repo
+helm repo add brigade https://brigadecore.github.io/charts
+# install Brigade
+helm install -n brigade brigade/brigade
+
+# if you want to activate Generic Gateway, you should use this command
+# helm install -n brigade brigade/brigade --set genericGateway.enabled=true
+```
+
+You will now have Brigade installed. [Kashti](https://github.com/brigadecore/kashti), the dashboard for your Brigade pipelines, is also installed in the cluster.
+
+## Install brig
+
+Brig is the Brigade command line client. You can use `brig` to create/update/delete new brigade Projects, run Builds, etc. To get `brig`, navigate to the [Releases page](https://github.com/brigadecore/brigade/releases/) and then download the appropriate client for your platform. For example, if you're using Linux or WSL, you can get the 1.0.0 version in this way:
+
+```bash
+wget -O brig https://github.com/brigadecore/brigade/releases/download/v1.0.0/brig-linux-amd64
+chmod +x brig
+mv brig ~/bin
+```
+
+Alternatively, you can use [asdf-brig](https://github.com/Ibotta/asdf-brig) to install & manage multiple versions of `brig`.
+
+We have two quickstarts for you to check.
```

Comment:
nit: could you use a colon, here?

Also, an empty line would be great -- see the rendering from the docs.

Reply:
fixing